tutorials/README.md

```diff
@@ -43,7 +43,8 @@ Once you have familiarised yourself with the methods and identified how to apply
 
 ### Analysis:
```

Comment:
I guess all of these will be filled with a short description in a later pull request ?

Reply:
yes exactly this is the plan!
Great thanks!